src/components/NpsSurvey.js

```diff
@@ -46,10 +46,16 @@ export const NpsSurvey = _.flow(
   }
 
   async loadStatus() {
-    const lastResponse = (await Ajax().User.lastNpsResponse()).timestamp
-    const oneMonthAgo = _.tap(d => d.setDate(d.getMonth() - 1), new Date())
+    const lastResponse = await Ajax().User.lastNpsResponse()
 
-    this.setState({ requestable: !lastResponse || (new Date(lastResponse) < oneMonthAgo) })
+    const currTime = new Date()
+    const currTimeHours = Date.parse(currTime)/3600000
+
+    const askTheUser = lastResponse
+      ? currTimeHours - new Date(lastResponse).getTime()/3600000 >= 168
```

Comment:
For duration values like this, I find that writing out the factors is good documentation of intent. For 3600000, I can guess that there are 2 60s in there which then leaves 1000 so it must be milliseconds per hour (`60 * 60 * 1000`). 168 is less obvious. `7 * 24` would make the intent easier to understand.

Also, no need to worry about the calculation happening repeatedly. I can't imagine there being a modern language/runtime that would not recognize this as a constant and calculate it exactly once.

Another thought... since you're doing `/3600000` in many places, you could extract a function for it which gives you a chance to give it a meaningful name, e.g. `millisToHours`.